Objects/Converters/ConverterRevit/ConverterRevitShared/ConversionUtils.cs

```diff
@@ -67,7 +67,7 @@ private bool ShouldConvertHostedElement(DB.Element element, DB.Element host, Bas
         var elementId = element.Id;
         if (!hostedElementIds.Contains(elementId))
         {
-          extraProps["speckleHost"] = new Base() { applicationId = host.UniqueId, ["category"] = host.Category.Name, };
+          extraProps["speckleHost"] = new Base() { applicationId = host.UniqueId, ["category"] = host.Category.Name };
```

Comment:
maybe should pass the `builtInCategory` prop here too?

Reply:
yes I think so, pass both the `category` and `builtInCategory` props